ngrinder-controller/src/main/java/org/ngrinder/script/repository/FileEntryRepository.java

```diff
@@ -132,29 +130,25 @@ public File getUserRepoDirectory(User user) {
 		SVNClientManager svnClientManager = getSVNClientManager();
 		try {
 			svnClientManager.getLogClient().doList(SVNURL.fromFile(getUserRepoDirectory(user)).appendPath(path, true),
-					svnRevision, svnRevision, true, recursive, new ISVNDirEntryHandler() {
-				@Override
-				public void handleDirEntry(SVNDirEntry dirEntry) throws SVNException {
-
-					FileEntry script = new FileEntry();
-					// Exclude base path "/"
-					if (StringUtils.isBlank(dirEntry.getRelativePath())) {
-						return;
-					}
-					script.setPath(FilenameUtils.normalize(path + "/" + dirEntry.getRelativePath(), true));
-					script.setCreatedDate(dirEntry.getDate());
-					script.setLastModifiedDate(dirEntry.getDate());
-					script.setDescription(dirEntry.getCommitMessage());
-					script.setRevision(dirEntry.getRevision());
-					if (dirEntry.getKind() == SVNNodeKind.DIR) {
-						script.setFileType(FileType.DIR);
-					} else {
-						script.getFileType();
-						script.setFileSize(dirEntry.getSize());
-					}
-					fileEntries.add(script);
-				}
-			});
+					svnRevision, svnRevision, true, recursive, dirEntry -> {
+						FileEntry script = new FileEntry();
+						// Exclude base path "/"
+						if (StringUtils.isBlank(dirEntry.getRelativePath())) {
+							return;
+						}
+						script.setPath(FilenameUtils.normalize(path + "/" + dirEntry.getRelativePath(), true));
+						script.setCreatedDate(dirEntry.getDate());
+						script.setLastModifiedDate(dirEntry.getDate());
+						script.setDescription(dirEntry.getCommitMessage());
+						script.setRevision(dirEntry.getRevision());
+						if (dirEntry.getKind() == SVNNodeKind.DIR) {
+							script.setFileType(FileType.DIR);
+						} else {
+							script.getFileType();
```

Comment:
Check if this is necessary

Reply:
It sets fileType if null.

https://github.com/naver/ngrinder/blob/develop/ngrinder-controller/src/main/java/org/ngrinder/script/model/FileEntry.java#L112